release/example/str.trie/example.str.trie.model_info

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Example SENĆOŦEN Trie Model",
+    "license": "mit",
+    "languages": [
+        "str",
+        "saan1246"
```

Comment:
[SENĆOŦEN doesn't have an ISO 639-3 code](https://en.wikipedia.org/wiki/Saanich_dialect), but it does have a Glottolog code. So I wrote that instead. I'm not sure what to do in cases like these, especially since SENĆOŦEN, despite being called a "dialect", has its own distinctive ALL-CAPS LATIN ORTHOGRAPHY.

Reply:
fwiw, the fv_sencoten keyboard is using `str-Latn` for the BCP 47 code

Reply:
We can only take a BCP 47 code here, which means we are limited to ISO639-1 and ISO639-3. We can use extensions but there are limits to those as well because of operating system limitations. Ideally we'd use something like `str-x-sencoten`; but no keyboard will match that. The SENĆOŦEN keyboard has the code `str` so let's work with that for now. This is the sort of ambiguity we can cover into the model name as well, I think (e.g. `example.str.trie_SENCOTEN` or something like that?)

Reply:
> The SENĆOŦEN keyboard has the code str so let's work with that for now. This is the sort of ambiguity we can cover into the model name as well, I think (e.g. example.str.trie_SENCOTEN or something like that?)

That sounds reasonable to me!

Reply:
Just tried this, and realized that we restricted the `uniq` part to be lowercase. Because I don't want freaky things happening on case-insensitive filesystems, I think `example.str.trie_sencoten` will be good enough for now.

Reply:
Following up on this. 

Can we include the language tag `str-Latn` to this model? I tried downloading the fv_sencoten keyboard **Salish, Straits (Latin)** but it doesn't find this associated model

Query is https://api.keyman.com/model?q=bcp47:str-Latn